envs/example/ci-full/group_vars/all.yml

```diff
@@ -1,17 +1,33 @@
 ---
-stack_env: example_ci-full
+stack_env: ci-full
+ursula_os: 'ubuntu'
+undercloud_cidr:
+  - cidr: 192.168.0.0/22
 
 state_path_base: /opt/stack/data
 
-neutron:
-  enable_external_interface: True
-  lbaas:
-    enabled: True
+
+hostname_fqdn: "{{ ansible_fqdn }}"
+hostname_short: "{{ hostname_fqdn.split('.')|first }}"
+
+floating_ip: "{{ hostvars[groups['controller'][0]][primary_interface]['ipv4']['address'] }}"
 
 keystone:
   ldap_domain:
     enabled: True
     domain: users
 
+neutron:
+  enable_external_interface: True
+  path_mtu: 1400
```

Comment:
It is in ci-full so it won't have impact to any environments except VMs based environments.

Why I set path_mtu to 1400 for ci-full is that I can ping, but won't be able to ssh VMs when VMs locate on compute. 

I checked ci job log and found that why we won't hit the problem in our ci for 31x. It is because that it skipped the testings to ssh VMs including "wait for instance to boot" and "test instance can ping Google". Both are enabled on master.

https://jenkins.ci.blueboxgrid.com/job/Ursula_Test_Ubuntu_Branch_PRs/37/consoleFull